src/plugins/data/public/index.ts

```diff
@@ -20,6 +20,8 @@
 import { PluginInitializerContext } from '../../../core/public';
 import { DataPublicPlugin } from './plugin';
 
+export * from './types';
```

Comment:
I think we should be explicit with exports at the top level

Reply:
Those are just TypeScript types, but done, I'm exporting now explicitly.